frontend/views/partials/genre.html

```diff
@@ -4,8 +4,42 @@
 	<title> Choose Genre</title>
 	<!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
 	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap-theme.min.css"> -->
+	<meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
 </head>
-<body style="margin-top: 300px">
+<body class="page-top" class="index">
+   <section id="portfolio" >
```

Comment:
@MinaNemr Just add documentation to the html genre.html, Just for things like where does each section start and end, for example like in index: <!-- start of dashboard --> , then after the closing tag <!-- end of dashboard --> things like this. just a general description of what is in each section and where does it end.